tests/testthat/test-bnma_analysis.R

```diff
@@ -122,7 +122,7 @@ test_that("BaselineRiskNetwork() assigns model type, covariate type and referenc
   bnma_network <- BaselineRiskNetwork(br_data = data,
                                       outcome_type = "Binary",
                                       effects_type = "random",
-                                      cov_parameters = "independent")
+                                      cov_parameters = "unrelated")
```

Comment:
gemtc and bnma use different words. BaselineRiskNetwork( ) originally used bnma language, so you could pass the parameter directly to bnma::network.data. Now it uses gemtc language, so "unrelated" has to be changed to "independent" before passing to bnma::network.data. The change is so the language is consistent for users, with shared/exchangeable/unrelated in both models rather than shared/exchangeable/unrelated in one and common/exchangeable/independent in the other.